weed/server/raft_hashicorp.go

```diff
@@ -25,7 +73,7 @@ func NewHashicorpRaftServer(option *RaftServerOption) (*RaftServer, error) {
 	}
 
 	c := raft.DefaultConfig()
-	c.LocalID = raft.ServerID(s.serverAddr) // TODO maybee the IP:port address will change
+	c.LocalID = raft.ServerID(s.serverAddr.String()) // TODO maybee the IP:port address will change
```

Comment:
Need to use the raw string instead of String() function. 
String() returns server + http port only.